post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+:context: creating-multi-arch-compute-nodes-gcp
+[id="creating-multi-arch-compute-nodes-gcp"]
+= Creating a cluster with multi-architecture compute machines on GCP
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+To create a Google Cloud Platform (GCP) cluster with multi-architecture compute machines, you must first create a single-architecture GCP installer-provisioned cluster with the multi-architecture installer binary. For more information on AWS installations, refer to xref:../../installing/installing_gcp/installing-gcp-customizations.adoc[Installing a cluster on GCP with customizations]. You can then add ARM64 compute machines sets to your GCP cluster.
+
+[NOTE]
+====
+Secure booting is currently not supported on ARM64 machines for GCP 
+====
+
+include::modules/multi-architecture-verifying-cluster-compatibility.adoc[leveloffset=+1]
```

Comment:
If this was a pre-existing module, please update its metadata to note that it has been included in this assembly.